lib/cli/src/generators/UPDATE_PACKAGE_ORGANIZATIONS/index.ts

```diff
@@ -54,7 +54,7 @@ function updateSourceCode(parser: string) {
   const jscodeshiftPath = path.dirname(require.resolve('jscodeshift'));
   const jscodeshiftCommand = path.join(jscodeshiftPath, 'bin', 'jscodeshift.sh');
 
-  ['update-organisation-name.js', 'move-buildin-addons.js'].forEach((codemod) => {
+  ['update-organisation-name.js', 'move-builtin-addons.js'].forEach((codemod) => {
```

Comment:
If you want to change this and the similar ones below, you need to rename the file too

Reply:
It is renamed: https://github.com/storybookjs/storybook/pull/12590/files#diff-1f7e4ef72821f3812b9000a0a8918760. GitHub unfortunately makes it very easy to miss such renames. (I often fail to catch such renames, so I appreciate the call out.)

Reply:
Hmm. Test is failing tho?

>     Cannot find module '/tmp/storybook/lib/codemod/src/transforms/move-buildin-addons' from 'lib/codemod/src/transforms/__tests__/transforms.tests.js'

Reply:
Oops, I renamed one file but not a related directory :-( fixing.